docs/content/guides/v4tov5.md

```diff
@@ -476,3 +476,17 @@ spec:
 ```
 
 Your upgrade is now complete! For more information on how to use PGO v5, we recommend reading through the [PGO v5 tutorial]({{< relref "tutorial/_index.md" >}}).
+
+## Additional Considerations
+
+If the upgrade to PGO v5 will also result in an OS upgrade from either CentOS 7 to CentOS 8, or
+from UBI 7 to UBI 8, you may need to perform additional steps in order to ensure a successful
+upgrade of your database.  For instance, changes to the GNU C library (`glibc`) in both CentOS 8
+and UBI 8 require you to reindex certain indexes _prior_ to the making the PostgreSQL cluster
+available for production use.
+
+Please see the following page on the [PostgreSQL Wiki](https://wiki.postgresql.org/) for more
+information about these additional `glibc` considerations when migrating between OS versions,
+along with any additional steps required to ensure a successful upgrade of your database:
+
+https://wiki.postgresql.org/wiki/Locale_data_changes
```

Comment:
```suggestion
## Additional Considerations

Upgrading to PGO v5 will result in an base image upgrade from EL-7 (UBI / CentOS) to EL-8 (UBI / CentOS). Based on the contents of your Postgres database, you may need to perform additional steps.

Due to changes in the GNU C library (`glibc`) in EL-8, you may need to reindex certain indexes in your Postgres cluster. For more information, please read the [PostgreSQL Wiki on Locale Data Changes](https://wiki.postgresql.org/wiki/Locale_data_changes), how you can determine if your indexes are affected, and how to fix them.
```

Reply:
I can update accordingly.  However, not clear on the issue with my initial submission.

Reply:
The suggested change has been added.